packages/devtools-launchpad/bin/development-server.js

```diff
@@ -35,18 +35,26 @@ function httpOrHttpsGet(url, onResponse) {
   });
 }
 
+function getFavicon() {
+  let favicon = getValue("favicon");
+
+  if (!favicon) {
+    return "launchpad-favicon.png";
+  }
+
+  return path.basename(favicon);
+}
+
 function serveRoot(req, res) {
   const tplPath = path.join(__dirname, "../index.html");
   const tplFile = fs.readFileSync(tplPath, "utf8");
 
-  let favicon = getValue("favicon");
   res.send(Mustache.render(tplFile, {
     isDevelopment: isDevelopment(),
     dir: getValue("dir") || "ltr",
+    bundleName: getValue("title").toLocaleLowerCase() || "bundle",
```

Comment:
I was thinking `getValue("title")` is undefined if there isn't a title entry in the configuration, hence, doing `.toLocaleLowerCase()` on undefined would have throw.
I'll check and open a PR if there's such issue